### ISSUE TITLE
not able to use all 64 cores on 1.24xlarge for resnet50 pytorch

### ISSUE THREAD

**Issue by zqliu19** [aws-neuron/aws-neuron-sdk#345]
num_neuron_cores = 64
nc_env = ','.join(['1'] * num_neuron_cores)
os.environ['NEURONCORE_GROUP_SIZES'] = nc_env

batch_total = num_neuron_cores * 5 * 1

batch_image = torch.zeros([batch_total, 3, 224, 224], dtype=torch.float32)
print(batch_image.shape)

## Load model
COMPILED_MODEL = 'resnet50_neuron_bs5_dynamic.pt'
BATCH_PER_THREAD = batch_total / num_neuron_cores
model_neuron = NeuronSimpleDataParallel(COMPILED_MODEL,num_neuron_cores,BATCH_PER_THREAD)

## Since the first inference also load the model let's exclude it from timing
results = model_neuron( batch_image )
print('warmup done!')
## Predict for 10 loops
loops = 10
latency = [None] * loops
for i in range(loops):
    start = time()
    results = model_neuron( batch_image )
    latency[i] = time()-start
print(results.shape)

## neuron-cli list-ncg shows 64 NCGs being created (which is expected)
Total NeuronDevices: 16 NeuronCores: 64 NCGs: 64
## BUT, neuron-top shows only  37 cores being used (sometimes it shows 38 or 41)
Models: 37 loaded, 0 running. NeuronCores: 37 used.
## re-run may have different cores being used, see below (41 instead of 37)
Models: 41 loaded, 0 running. NeuronCores: 41 used.
## This looks like a NEURON bug, could you help take a look, thx!
